### ISSUE TITLE
Repo tutorials do not match online tutorials

### ISSUE THREAD

**Issue by DanRathbun** [SketchUp/sketchup-ruby-api-tutorials#2]
Repo tutorials do not match online tutorials. **One or the other is behind.** 

The repo has one more example than online, and two more tutorials than online developer site. 

_Ref:_ http://developer.sketchup.com/en/content/tutorials

**Comment by thomthom** [comment]
Yes, the developer site is behind. Though we didn't intent to make tutorials of everything we post in the github repo. The developer site would have a more curated set of tutorials for  on-boarding, where as this repo was intended as a general collection of examples and simple code-tutorials.

Still working out the format of this.

**Comment by DanRathbun** [comment]
> @thomthom: Still working out the format of this.

Yes, I was expecting them to be divided into categories. 
Instead there is some number prefixes.

I'd think the numbered "introductory course" should be in a category of that (or a similar) name. Then we have other categories, such as "Observers" and "Collections", etc.
